projects/plugins/boost/app/modules/optimizations/cloud-css/Cloud_CSS.php

```diff
@@ -51,6 +52,15 @@ public function setup() {
 		return true;
 	}
 
+	/**
+	 * Check if the module is ready and already serving critical CSS.
+	 *
+	 * @return bool
+	 */
+	public function is_ready() {
+		return ( new Critical_CSS_State() )->is_generated();
+	}
```

Comment:
If this is called `ready` perhaps the above "get_active_optimizatoin_modules` should be `get_ready_optimizations` ? 

I feel like this PR is introducing `ready` as a concept, so the name should be used consistently.